fixtures/test-readFile/hello-world.yaml

```diff
@@ -0,0 +1 @@
+hello-world
```

Comment:
It is best if files end in a newline (unless you're deliberately testing the behavior when it's missing).

Reply:
Much better -- it looks much more like YAML now.  😉